source/frontend/src/features/documents/list/DocumentListContainer.tsx

```diff
@@ -48,6 +49,7 @@ const DocumentListContainer: React.FunctionComponent<
   const onDelete = async (
     documentRelationship: Api_DocumentRelationship,
   ): Promise<boolean | undefined> => {
+    props.onSuccess?.();
```

Comment:
yeah, this might be the issue, this seems to run before the actual delete request.

Reply:
good catch. I updated the notes one and this one slipped by